src/vivarium_census_prl_synth_pop/tools/cli.py

```diff
@@ -57,25 +57,15 @@ def make_artifacts(
 
 
 @click.command()
-@click.argument("output_file", type=click.Path(exists=True))
+@click.argument("output_dir", type=click.Path(exists=True))
 @click.option("-v", "verbose", count=True, help="Configure logging verbosity.")
 @click.option(
     "--pdb",
     "with_debugger",
     is_flag=True,
     help="Drop into python debugger if an error occurs.",
 )
-@click.option(
-    "-s",
-    "--single",
-    "single_run",
-    default=False,
-    is_flag=True,
-    help="Results are from a single, non-parallel run.",
-)
```

Comment:
Is this removed because we are not planning to support parallel runs in the future, or just because they don't work _yet_ due to the observer overwriting issue? If the latter, would it make sense to leave this and raise a NotImplementedError or something in build_results?

Reply:
This is removed because I don't want the output of a single sim to be any different from a parallel sim that only has a single seed.

Reply:
That makes sense, but what about the output of a parallel sim with _multiple_ seeds? That will need to be handled here eventually, right?

Reply:
This currently is broken in the simulaton (since each parallel run tries to write to the same location). That needs to be fixed and I think a ticket has bee created for that. When that is fixed (as part of that ticket), this post-processing work will have to be updated.